src/components/organisms/DataExplorer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Pane } from 'evergreen-ui';
+import DataResultItem from '../molecules/DataResultItem';
+import { useQuery } from '@apollo/client';
+import { METADATA, MetadataEntry } from '../../queries/metadata';
+import Loading from '../atoms/Loading';
+import ErrorMessage from '../atoms/ErrorMessage';
+
+const DataExplorer: React.FC = () => {
+    const { data, loading, error } = useQuery(METADATA);
```

Comment:
useQuery should take in the metadata type, because typescript 😃 ```useQuery<T>(METADATA)```

Reply:
Smart 👯